arm-apimanagement/2016-07-07/swagger/apimanagement.json

```diff
@@ -6321,6 +6597,150 @@
         "type"
       ],
       "description": "Parameters supplied to the Update Logger operation."
+    },    
+    "BackendGetContract": {
```

Comment:
I'm still confused about the names of these classes.
`BackendGetContract` sounds like an operation ("Get"?) has found its way into this type name - what does the "Get" mean? Is this a response model? In that case, rather call it "BackendContractResponse"
Having no background about this API, I think the name of a model should be understandable after reading the description.
Likewise, I still have no clue why a `BackendPaged` consists of `BackendGetContract`s.
Is there any way this can be made more intuitive, e.g. the regular "XyzCollection" convention?